htdocs/core/class/html.form.class.php

```diff
@@ -4742,6 +4742,41 @@ public function select_types_paiements($selected = '', $htmlname = 'paiementtype
 		}
 	}
 
```

Comment:
the two following functions have been written in CamelCase because it's best practice, but they're based on existing functions that are in snake_case. I can change if coherence is more important.